docs/continuous-delivery/gitops/secret-management/secret-management-gitops.md

```diff
@@ -0,0 +1,177 @@
+# Secret Management with GitOps
+
+GitOps primarily have git repositories in the center considered a source of truth for managing both infrastructure and application. This Infrastructure and application will require secured access to other resources of the system through secrets. Committing clear text secrets into git repositories is unacceptable even if the repositories are private to your team and organization. Teams need a secure way to handle secrets when using GitOps.
+
+There are many ways to manage secrets with GitOps and at high level can be categorized into:
+
+1. Encrypted Secrets in Git Repositories
+1. Reference to secrets stored in the external key vault
+
+## 1. Encrypted Secrets in Git Repositories
+
+In this approach, secrets are manually encrypted by developers using a public key and the custom Kubernetes controller running in the target cluster can only decrypt the key. Some popular tools for his approach are [Bitnami Sealed Secrets](https://github.com/bitnami-labs/sealed-secrets), [Mozilla SOPS](https://github.com/mozilla/sops)
+
+- Secret changes are managed by making changes within the GitOps repository which provides great traceability of the changes made
```

Comment:
It may be useful to include a sentence leading into these bullets to provide context, such as "Benefits of using this approach include:"